pkg/apis/core/validation/validation.go

```diff
@@ -1190,6 +1193,9 @@ func validateRBDVolumeSource(rbd *core.RBDVolumeSource, fldPath *field.Path) fie
 	if len(rbd.RBDImage) == 0 {
 		allErrs = append(allErrs, field.Required(fldPath.Child("image"), ""))
 	}
+	if rbd.Keyring != "" && !filePathRegex.MatchString(rbd.Keyring) {
```

Comment:
same here, tightening validation causes problems in the API. can validate it at point of use